### DISCUSSION TITLE
Temperature stuck @ 85c

### DISCUSSION THREAD

**Discussion by fabiobarbosa1984** [BernhardSchlegel/BierBot-Bricks#14] [Q&A] [answered]
Hi folks!

I don't know if I did something wrong while flashing the sonoff, but my temperature are stucked at 85c. If I disconnect the sensor, then it goes to 0c. This area the only two readings that I got. 

I google it and found some problem related to the original firmware that stucks the temperature at 85c. But it doesn't make sense since I using the bierbot bricks firmware. 

Is there any way to reset the brick o another thing that I could do to try to fix it?

**Comment by BernhardSchlegel** [comment]
Hey Fabio, thanks for reaching out ! Sorry to hear about your issues. If the problem exists across multiple, completely different firmwares (Sonoff and BierBot) that sounds to me like a hardware issue. Easiest way would be to try a different temperature probe. 

Also, I can offer to take a look if have a issue backend-wise. For that you would need to invite me to your brewery ("Bernhard") and tell me the name of your brewery.

Please update this thread if you figure out the solution, other will profit from that. I'll do the same!

Thanks,
Bernhard.

**Comment by fabiobarbosa1984** [comment, marked as answer]
I found a solution! The problem was bad construction of the sensor. I openned the sonoff and pressed hard the metal parts of the temperature connector. After, i just powered the brick and "voilà"! Got the temperature reading!

**Reply by BernhardSchlegel** [reply]
Glad to hear that :)
